packages/arb-token-bridge-ui/src/components/TransferPanel/USDCDeposit/USDCDepositConfirmationDialog.tsx

```diff
@@ -0,0 +1,178 @@
+            </button>
+          </div>
+
+          <Tab.List className="bg-ocl-blue">
+            {isArbitrumOne && <TabButton>Use a third-party bridge</TabButton>}
+            <TabButton>Use Arbitrum’s bridge (USDC.e)</TabButton>
+            <TabButton>Use Arbitrum’s bridge (USDC)</TabButton>
+          </Tab.List>
+
+          {isArbitrumOne && tokenSymbol && (
+            <Tab.Panel className="flex flex-col space-y-3 px-8 py-4">
+              <div className="flex flex-col space-y-3">
+                <p className="font-light">
+                  Receive{' '}
+                  <ExternalLink
+                    className="arb-hover text-blue-link underline"
+                    href={`https://arbiscan.io/token/${CommonAddress.ArbitrumOne.USDC}`}
+                  >
+                    USDC
+                  </ExternalLink>{' '}
+                  on Arbitrum One using a third-party bridge with Circle’s CCTP
+                  integrated.
+                </p>
+              </div>
+
+              <BridgesTable
+                bridgeList={fastBridges}
+                selectedNonCanonicalToken={tokenSymbol}
+              />
+              <div className="mt-2 flex flex-row justify-end space-x-2">
+                <Button
+                  variant="secondary"
+                  onClick={() => props.onClose(false)}
+                >
+                  Cancel
+                </Button>
+              </div>
+            </Tab.Panel>
+          )}
+
+          {tokenSymbol && (
+            <Tab.Panel className="flex flex-col space-y-3 px-8 py-4">
+              <div className="flex flex-col space-y-3">
+                <p className="font-light">
+                  Receive{' '}
+                  <ExternalLink
+                    className="arb-hover text-blue-link underline"
+                    href={`https://arbiscan.io/token/${CommonAddress.ArbitrumOne['USDC.e']}`}
+                  >
+                    Bridged USDC (USDC.e)
+                  </ExternalLink>{' '}
+                  on Arbitrum One using Arbitrum’s native bridge.
+                </p>
+
+                <div className="flex flex-col space-y-6">
+                  <USDCDepositConfirmationDialogCheckbox
+                    checked={USDCcheckboxChecked}
+                    onChange={setUSDCcheckboxChecked}
+                  />
+                </div>
+              </div>
+
+              <div className="mt-2 flex flex-row justify-end space-x-2">
+                <Button
+                  variant="secondary"
+                  onClick={() => props.onClose(false)}
+                >
+                  Cancel
+                </Button>
+                <Button
+                  variant="primary"
+                  disabled={!USDCcheckboxChecked}
+                  onClick={() => {
+                    props.onClose(true)
+                    setUSDCcheckboxChecked(false)
+                    trackEvent('Use Arbitrum Bridge Click', {
```

Comment:
We also have `USDC: Use Arbitrum Bridge Click`. Which one do we use? Do we need to list both?

Reply:
Oh, and when do we actually fire `USDC: Fast Bridge Click` events?

Reply:
it's converted here
https://github.com/OffchainLabs/arbitrum-token-bridge/blob/450fc9d74dd29530751eab973f8fdd4cf95877a0/packages/arb-token-bridge-ui/src/util/AnalyticsUtils.ts#L338

https://github.com/OffchainLabs/arbitrum-token-bridge/blob/450fc9d74dd29530751eab973f8fdd4cf95877a0/packages/arb-token-bridge-ui/src/util/AnalyticsUtils.ts#L300-L304

Reply:
Fast Bridge Click is here
https://github.com/OffchainLabs/arbitrum-token-bridge/blob/450fc9d74dd29530751eab973f8fdd4cf95877a0/packages/arb-token-bridge-ui/src/components/common/BridgesTable.tsx#L28-L31

Reply:
Events starting with `USDC:` are Fathom events. Do we even want those in?

Reply:
i didn't know we are not doing fathom events when started this PR
doesn't matter if they are in this PR with empty event codes

checked with PM that the posthog events with properties look fine